### PR TITLE
docs: add securing TiKV cluster doc

### PR DESCRIPTION
**Pull request by CaitinChen**
Signed-off-by: CaitinChen <chenchun@pingcap.com>

## What have you changed? (mandatory)

Add the securing TiKV cluster doc

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

Relevant

### CODE REVIEW

**Comment by CaitinChen** [comment]
@overvenus @lilin90 PTAL

File: docs/secure-tikv-cluster.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Secure the TiKV Cluster
```
Comment (lilin90) [inline docs/secure-tikv-cluster.md]: -> TiKV Security Configuration
(I think readers might expect a step-by-step procedure from the previous title.)

File: docs/secure-tikv-cluster.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Secure the TiKV Cluster 
+summary: Learn how to secure the TiKV cluster.
+category: operations
+---
+
+# Secure the TiKV Cluster
```
Comment (lilin90) [inline docs/secure-tikv-cluster.md]: Ditto

File: docs/secure-tikv-cluster.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Secure the TiKV Cluster 
+summary: Learn how to secure the TiKV cluster.
```
Comment (lilin90) [inline docs/secure-tikv-cluster.md]: -> Learn about the security configuration in TiKV.

File: docs/secure-tikv-cluster.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+category: operations
+---
+
+# Secure the TiKV Cluster
+
+TiKV has SSL/TLS integration to encrypt the data exchanged between nodes. This document describes how to secure the TiKV cluster.
```
Comment (lilin90) [inline docs/secure-tikv-cluster.md]: how to secure -> the security configuration in

**Comment by CaitinChen** [comment]
@lilin90 PTAL Thanks

File: docs/secure-tikv-cluster.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Secure the TiKV Cluster 
-summary: Learn how to secure the TiKV cluster.
+title: TiKV Security Configuration 
+summary:  Learn about the security configuration in TiKV.
```
Comment (lilin90) [inline docs/secure-tikv-cluster.md]: Please delete the extra space before "Learn".

**Comment by CaitinChen** [comment]
@overvenus PTAL

**Pull request merged**